tests/testthat/test-bm-tpc-h.R

```diff
@@ -83,13 +88,25 @@ ORDER BY
     l_linestatus;\n"
   )
 
-  # create a new connection to ensure we're using the one that is bieng based
+  # create a new connection to ensure we're using the one that is being based
   con_one <- DBI::dbConnect(duckdb::duckdb())
-  DBI::dbExecute(con_one, paste0("CALL dbgen(sf=0.001);"))
+
+  # use it to populate the tables needed for query 1
+  get_input_func(
+    engine = "duckdb",
+    scale_factor = 0.001,
+    format = "parquet",
+    query_id = 1,
+    con = con_one
+  )
 
   query_01_func <- get_sql_query_func(1)
-  query_01_func(con = con_one)
+  expect_s3_class(query_01_func(con = con_one), "data.frame")
 
   # clean up
   DBI::dbDisconnect(con_one, shutdown = TRUE)
-})
\ No newline at end of file
+})
+
+test_that("tpch_answer() works", {
+
+})
```

Comment:
It looks like this can be removed? You have the test up above that does both of the options here, yeah?